docs/release/release_0_3_0.md

```diff
@@ -48,19 +48,19 @@ allow users to get help with any issues they might have using napari.
 
 ### IO plugins
 
-napari is now extensible to open and save to a wide variety of file formats,
-both local and remote. Currently, the same file formats as before are available
+napari now supports opening and saving in a wide variety of file formats,
+both local and remote, through an extensible plugin architecture. The same file formats as before are available
```

Comment:
@sofroniewn I'm not sure I like this wording as much, because out of the box we don't really support many more formats. The point is that we *can* support many more formats...

Reply:
"now will be able to support"

Reply:
How about "Contributors can now easily extend napari to open and save in a wide variety..."?

Reply:
Good to me, will fix once you finish this review

Reply:
done